src/plugins/syscalls/syscalls.cpp

```diff
@@ -149,6 +227,27 @@ static GSList* create_trap_config(drakvuf_t drakvuf, syscalls *s, symbols_t *sym
 
             PRINT_DEBUG("[SYSCALLS] Adding trap to %s\n", symbol->name);
 
+            syscall_wrapper_t *wrapper = (syscall_wrapper_t *)g_malloc(sizeof(syscall_wrapper_t));
+
+            wrapper->syscall_index = -1;
+
+            for (j=0; j<NUM_SYSCALLS; j++) {
+              if(strcmp(symbol->name,win_syscall_struct[j].name)==0) {
+                wrapper->sc=s;
+                wrapper->syscall_index=j;
+              }
+            }
+
+            // These functions are not found:
+            // ERROR: NtGetEnvironmentVariableEx not found
+            // ERROR: NtQueryEnvironmentVariableInfoEx not found
+            if(wrapper->syscall_index==-1) {
+              printf("ERROR: %s not found\n",symbol->name);
+
+              wrapper->sc=s;
+              wrapper->syscall_index=0; //we'll just return garbage so at least we won't crash
```

Comment:
Syscall index `0` is a valid though. IMHO better to keep it as -1 and check in the callback if we have a valid index and if not, just skip the argument extraction.